tests/integration/scripts/test_copy_terraform_built_artifacts.py

```diff
@@ -11,7 +11,7 @@
 
 class TestCopyTerraformBuiltArtifacts(TestCase):
     def setUp(self) -> None:
-        self.script_dir = pathlib.Path(__file__).parents[3].joinpath("samcli").joinpath("scripts")
+        self.script_dir = pathlib.Path(__file__).parents[3].joinpath("samcli/hook_packages/terraform")
```

Comment:
I don't know if this will work on windows too because of the different path separators on windows. I'm going to approve this, but check on the gates to ensure.

Reply:
I realized I used joinpath wrong, and just had to provide the different parts of the path separated by commas. Fixed now